packages/common/src/shared-types.ts

```diff
@@ -283,6 +284,11 @@ export interface AdminUiConfig {
      * An array of languages for which translations exist for the Admin UI.
      */
     availableLanguages: LanguageCode[];
+    /**
+     * @description
+     * An array of locales to be used on Admin UI.
+     */
```

Comment:
This will go in the next minor release, so add this:
```
@since 2.2.0
```

Reply:
done ✅